source/docs/articles/organizations/saml-for-orgs.md

```diff
@@ -0,0 +1,32 @@
+---
+title: SAML for Orgs
+description: Learn about single-sign on.
+category:
+  - developing
+---
+
+Single-sign on allows your users to authenticate against your IdP when using the Pantheon product.

```

Comment:
"Single-sign on" should be "Single sign-on (SSO)"

source: http://en.wikipedia.org/wiki/Single_sign-on